docs/start/quickstart.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Quickstart
+position: 2
+---
+# Quickstart
+
+Infra is rolling out it's Software as a Service offering which makes it easier than ever to setup infrastructure access for your organization. This page will show you everything you need to do to get started.
```

Comment:
These two paragraphs are a ton of frontmatter that may not help the user get started. Let's help them get started!

Reply:
refactored

Reply:
take a look at #3270